manager.go

```diff
@@ -86,18 +84,9 @@ func NewManager(dataDir string) (*Manager, error) {
 		}
 	}
 
-	amgr.wg.Add(1)
-	go amgr.addressHandler()
```

Comment:
Unless I'm missing something, I don't see where this is being run now.  It's not in the `run` func and it's no longer waiting for its shutdown either.

Reply:
`addressHandler` was renamed to `run`

Reply:
Oh, I saw the removal of the defer and missed the rename.  Thanks.